repose-aggregator/src/docs/asciidoc/recipes/docker.adoc

```diff
@@ -102,7 +110,7 @@ Let's break that command down and take a closer look at what it is doing:
     See our http://www.openrepose.org/versions/latest/release-notes.html[Release Notes] page for a list of available versions.
 <3> This option defines the tag to apply to the image.
     It is in `repository:tag` format.
-<4> The last parameter is the path to the *Repose* Dockerfile.
+<4> The last parameter is the directory that contains the Dockerfile.
```

Comment:
Heh, technically it's the path to the directory that contains the Dockerfile. The Docker man page just refers to it as path, though:
> Usage:	docker build [OPTIONS] PATH | URL | -`

Reply:
Updated.